tests/translator/input/connector_bucket_to_function.yaml

```diff
@@ -12,7 +12,7 @@ Resources:
 
   Bucket:
     # See also https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket-notificationconfig.html
-    DependsOn: [MyConnectorWriteLambdaPermission]
+    DependsOn: MyConnector
```

Comment:
nit: This should still be [MyConnector] right?

Reply:
Not necessarily, both are correct: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-dependson.html